shell/composables/useLabeledFormElement.ts

```diff
@@ -70,7 +70,9 @@ export const labeledFormElementProps = {
   }
 };
 
-export const useLabeledFormElement = (props: LabeledFormElementProps, emit: (event: string, ...args: any[]) => void): UseLabeledFormElement => {
+export type emitMethodType = (event: string, ...args: any[]) => void;
+
+export const useLabeledFormElement = (props: LabeledFormElementProps, emit: emitMethodType): UseLabeledFormElement => {
```

Comment:
I think that we might be able to more explicitly type the emits by using `defineEmits`:

```suggestion
const labeledFormElementEmits = defineEmits(['update:validation']);

export const useLabeledFormElement = (props: LabeledFormElementProps, emit: typeof labeledFormElementEmits): UseLabeledFormElement => {
```

This should accomplish two things:

1. Simplify consumption of `useLabeledFormElement` by removing the requirement to type emits as `emitMethodType`
2. Enforces that consumers define `update:validation` in their emits

Reply:
Added, I had to also add the defineEmits import and remove the emitMethodType in the other file so I added it as a part of my commit instead of committing the suggestion.